apis/goharbor.io/v1beta1/exporter_types.go

```diff
@@ -56,6 +56,9 @@ type ExporterSpec struct {
 	// +kubebuilder:validation:Required
 	Database ExporterDatabaseSpec `json:"database"`
 
+	// +kubebuilder:validation:Required
+	JobService ExporterJobServiceSpec `json:"jobservice"`
```

Comment:
If this is a required field, there may be a problem going from v1alpha3 to v1beta1 and the initial value needs to be added to the convert